app/javascript/controllers/notification_controller.js

```diff
@@ -0,0 +1,87 @@
+import { Controller } from "@hotwired/stimulus"
+
+// Connects to data-controller="notification"
+export default class extends Controller {
+  static targets = ["notification", "newHappyThingform", "happyThings"]
+
+  connect() {
+    console.log("👻 buh from notifications");
+    console.log("newHappyThingform", this.newHappyThingformTarget);
+    console.log("happyThings", this.happyThingsTarget);
+    console.log("notification", this.notificationTarget);
```

Comment:
Normally you don't want console logs when everything is working. It will annoy you in the future when you're testing something else.